packages/wrangler/src/publish.ts

```diff
@@ -247,13 +247,15 @@ export default async function publish(props: Props): Promise<void> {
       : `/accounts/${accountId}/workers/scripts/${scriptName}`;
 
     // Upload the script so it has time to propagate.
-    const { available_on_subdomain } = await fetchResult(
+    const { available_on_subdomain } = await fetchResult<{
+      available_on_subdomain: boolean;
+    }>(
       workerUrl,
       {
         method: "PUT",
         body: createWorkerUploadForm(worker),
       },
-      new URLSearchParams({ available_on_subdomain: "true" })
+      new URLSearchParams({ include_subdomain_availability: "true" })
```

Comment:
I can confirm that this is the correct parameter (by looking at the internal code). But it is worrying that this does not appear in our API docs, as far as I can tell.

Reply:
It's not a documented api, I found it on an internal wiki, I don't think it ever went public.